### DISCUSSION TITLE
🗣 Announcement: Live event with Rockford Lhotka

### DISCUSSION THREAD

**Discussion by rockfordlhotka** [MarimerLLC/csla#1600] [General]
Join Rocky for an open online chat and discussion about the current and future state of CSLA .NET.

[May 14, 2020, 1600 US central time (4 pm)](https://store.lhotka.net/rockford-lhotka-online)

Come prepared with your #cslanet questions!

**Comment by mtavares628** [comment]
Are we still able to register for this?  I'm getting Page Not Found when using the link.

**Reply by rockfordlhotka** [reply]
Sorry about that, mistyped the end time in the store. It is available again.

**Comment by rockfordlhotka** [comment]
For everyone who registered for the event later today, I'll be sending meeting invites soon.

**Comment by rockfordlhotka** [comment]
If you registered for today's event, you should have a calendar invite. The invite was sent to the billing email address you used in the store.

If you don't have a calendar invite let me know ASAP so we can resolve that issue.

**Reply by mtavares628** [reply]
Hi Rocky, 
I didn't receive a calendar invite.  What do you need from me?

**Reply by rockfordlhotka** [reply]
@mtavares628 I just sent an email to everyone who's registered. I think the calendar invite didn't get sent, or perhaps got swallowed by spam filters.

Did you get an email?

**Reply by mtavares628** [reply]
I just got it. Thanks.

**Comment by JacoJordaan** [comment]
Hi Rocky,

I also did not receive a calendar invite.

**Reply by rockfordlhotka** [reply]
@JacoJordaan did you just receive an email?

**Reply by JacoJordaan** [reply]
Just received e-mail now, but it is not in a readable format. Outlook does not see it as an invite.

**Reply by rockfordlhotka** [reply]
I sent using a different email client, hopefully better?

**Reply by JacoJordaan** [reply]
Got it. Thanks

**Comment by rockfordlhotka** [comment]
I have added the event connection details as a PDF to the item in the store. So if you didn't get a calendar invite or email from me, you can download the PDF from the store.

**Comment by kcabral817** [comment]
This event was really good.

**Reply by Chicagoan2016** [reply]
> This event was really good.

Yes, absolutely gold, looking forward to the next event. We should open a thread for the topic(s) of next event. @rockfordlhotka, may be, cover a specific topic like Data portal configuration?
Kind Regards
